LondonBritishLibrary/orient/BLorient4930.xml

```diff
@@ -0,0 +1,400 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Psalter</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 4930</idno>
+                        <altIdentifier><idno>Strelcyn 28</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="5r" to="124r"/><locus from="125r" to="162rb"/>
+                            <title ref="LIT2701Dawit"/>
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="5r" to="124r"/>
+                                <title>Psalms</title>
+                                <note>The text is divided for the days of the week, preceded by an introductory supplication. The titles of the psalms are 
+                                    the new ones.</note>
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus target="#5r"/>
+                                    <title ref="LIT5884IntroHymnPs"/>
+                                    <incipit xml:lang="gez">እስመ፡ <title>መዝሙረ፡ ዳዊት፡ </title>ትመስል፡ ገነት።
+                                        አንተ፡ ታስተጋብእ፡ አቅማሐ፡ ፍሬያት።
+                                        ትሰድድ፡ አጋንንት፡ <handShift/>ለውርጃ፡ <sic>ጽፈሕ፡ </sic>በወገሻ፡ ቅሰር፡<handShift/>
+                                        ወታቀርብ፡ መላእክተ። ተግሣፅ፡ ለኵሱ፡
+                                        ፍካሬ፡ ዘጻድቃን፡ ወዘኃጥአን።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.1.2">
+                                    <locus from="5r" to="124r"/>
+                                    <title ref="LIT2000Mazmur"/>
+                                    <incipit xml:lang="gez"><title>መዝሙር፡ ዘዳዊት።</title> ሃሌ፡ ሉያ። </incipit>
+                                </msItem>
+                            </msItem>
```

Comment:
LIT4261Prayer (now item ms_i2) should be somewhere here if it is the same hand and the same unit (it looks that this happens also in other Psalters)

Reply:
Strelcyn says nothing about a hand shift at this point, so I assume it is the same hand. He also reports for the layout that both the obligatory and the additional parts of the Psalter up to 146v are written in one column. This supports the idea that the different parts were conceived as one.